security/dsa_rbac.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+---
```

Comment:
since it is a general service account, should we use the name as dsa_rbac.yaml? or can we use a general service account yaml that can share with the other features?

Reply:
we cannot do that as it has reference to scc we use and both QAT and DSA use different SCCs. So they need to be different